src/sage/matroids/database_matroids.py

```diff
@@ -765,12 +824,12 @@ def F8():
         ],
         4: ['abcdefgh'],
     }
-    M = CircuitClosuresMatroid(groundset='abcdefgh', circuit_closures=CC)
-    M.rename("F8: " + repr(M))
+    M = Matroid(groundset='abcdefgh', circuit_closures=CC)
+    M = _rename_and_relabel(M, "F8", groundset)
```

Comment:
why is passing `groundset` necessary here?

Reply:
In order to relabel the matroid's groundset in case the user specifies one.